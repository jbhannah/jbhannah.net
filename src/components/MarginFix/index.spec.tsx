import { shallow } from "enzyme"
import * as React from "react"
import { MarginFix } from "."

const TestMarginFix = (props = {}) => shallow(<MarginFix {...props} />)

describe("MarginFix", () => {
  it("renders correctly", () => {
    const tree = TestMarginFix()
    expect(tree).toMatchSnapshot()
  })
})
