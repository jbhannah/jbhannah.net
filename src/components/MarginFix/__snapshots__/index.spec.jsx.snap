// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MarginFix renders correctly 1`] = `
<span
  css={
    Object {
      "display": "none",
    }
  }
/>
`;
